doc/configuration.rst

```diff
@@ -74,6 +74,10 @@ Globals Command-line options
 
   * ``--consul_ssl_verify``: If set to false, disable Consul certificate checking (true by default is ssl enabled).
 
+.. _option_terraform_plugins_dir_cmd:
+
+  * ``--terraform_plugins_dir``: Specify the directory where to find Terraform pre-installed providers plugins. If no specified, required plugins will be downloaded during deployment. See https://www.terraform.io/guides/running-terraform-in-automation.html#pre-installed-plugins for more information.
```

Comment:
typo: If no specified -> If not specified

Reply:
In addition to a new section in CHANGELOG.md to describe the Yorc dependency on terraform version and constraints on plugins versions (see similar dependency section in changelog in 3.1.0-M2 for ansible),
Maybe we could add here as well the plugin versions constraints Yorc has on plugins, else the user would discover only later at usage he has downloaded a plugin version incompatible with Yorc hardcoded expectations